operator/controllers/seldondeployment_controller.go

```diff
@@ -54,13 +56,21 @@ import (
 const (
 	ENV_DEFAULT_ENGINE_SERVER_PORT      = "ENGINE_SERVER_PORT"
 	ENV_DEFAULT_ENGINE_SERVER_GRPC_PORT = "ENGINE_SERVER_GRPC_PORT"
+	ENV_AMBASSADOR_ENABLED              = "AMBASSADOR_ENABLED"
 	ENV_CONTROLLER_ID                   = "CONTROLLER_ID"
 
 	DEFAULT_ENGINE_CONTAINER_PORT = 8000
 	DEFAULT_ENGINE_GRPC_PORT      = 5001
 
-	AMBASSADOR_ANNOTATION = "getambassador.io/config"
-	LABEL_CONTROLLER_ID   = "seldon.io/controller-id"
+	AMBASSADOR_ANNOTATION      = "getambassador.io/config"
+	LABEL_CONTROLLER_ID        = "seldon.io/controller-id"
+	ISTIO_DEFAULT_INGRESS      = "istio-ingressgateway.istio-system.svc.cluster.local"
+	AMBASSADOR_DEFAULT_INGRESS = "ambassador.ambassador.svc.cluster.local"
```

Comment:
This is assuming ambassador is in ambassador namespace?

Reply:
Yeah that's one of the todos in the issue, but wanted to confirm whether we should add the functionality to extract location of ambassadro and istio ingress details (name, namespace, etc)

Reply:
I think you would need some settings/envars otherwise it would break for most people.

Reply:
Oh ok that's a good idea